packages/sns/lib/sns/AbstractSnsService.ts

```diff
@@ -85,6 +90,7 @@ export abstract class AbstractSnsService<
     this.snsClient = dependencies.snsClient
   }
 
+  // TODO: should we keep it public? TBD
```

Comment:
we can make it protected

Reply:
Sorry for the early PR, I wanted to check if the install error was due to something wrong on my machine or not 😓

Yeah, I thought that too, leave it open sounds like a good approach 😊
Thank you so much Igor 🙏🏼